### ISSUE TITLE
Cosmetics do not show in beta warning screen

### ISSUE THREAD

**Issue by Wesley51** [AsoDesu/IslandUtils#12]
I'm assuming this is because the cosmetics don't display yet. I think the simple solution would be to remove the display for the beta screen.

What it is:
![2022-09-12_19 20 28](https://user-images.githubusercontent.com/59321467/189775010-0c6027d0-dee9-4004-9d9b-471b31118be5.png)
What it should look like:
![2022-09-12_19 20 35](https://user-images.githubusercontent.com/59321467/189775053-c9ce6884-5ad6-4d6e-a288-23fb83ca944b.png)

**Comment by LukynkaCZE** [comment]
Yeah the cosmetics are not loaded until you close the beta screen. This should be easy fix by just disabling character preview in the beta screen

**Comment by Wesley51** [comment]
Okay. I've also noticed the mod is displaying cosmetics on other servers. I will make a full report soon once I have the details.
